articles/language/file-structure.md

```diff
@@ -3,7 +3,7 @@
 title: File Structure | Microsoft Docs 
 description: Q# File Structure
 author: QuantumWriter
-uid: microsoft.quantum.qsharp-ref.file-structure
+uid: microsoft.quantum.language.file-structure
```

Comment:
I think this change makes sense, but can break existing links; we should double-check that there's no existing links to `xref:microsoft.quantum.qsharp-ref.file-structure`.